backend/btrixcloud/operator.py

```diff
@@ -402,6 +402,8 @@ async def update_crawl_state(self, redis, crawl, status):
         if crawl.stopping:
             print("Graceful Stop")
             await redis.set(f"{crawl.id}:stopping", "1")
+            # backwards compatibility with older crawler
+            await redis.set("crawl-stop", "1")
```

Comment:
yeah, I think I added this in process of trying to resolve the test failure hoping it would help somehow.. can keep for now, and remove in a future release.